src/features/Mypage/MyPageHeader/MyPageHeader.module.scss

```
@@ -76,33 +99,53 @@
+    width: 100%;
+    min-width: 0;
+
+    @include m.mq(sm) {
+      flex: 1;
+      width: auto;
+    }
   }
 
   &__title {
     @include m.text-style(xl, 700);
     color: var(--text);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)